hypervisor/boot/sbl/multiboot.c

```diff
@@ -287,21 +207,22 @@ int init_vm_boot_info(struct vm *vm)
 	if ((mbi->mi_flags & MULTIBOOT_INFO_HAS_CMDLINE) != 0U) {
 		char *cmd_src, *cmd_dst;
 		uint32_t off = 0U;
-		void *boot_params_addr;
+		bool status = false;
 		char buf[MAX_BOOT_PARAMS_LEN];
 
 		cmd_dst = kernel_cmdline;
 		cmd_src = hpa2hva((uint64_t)mbi->mi_cmdline);
 
-		boot_params_addr = parse_image_boot_params(vm, cmd_src);
-		/*
-		 * Convert ImageBootParamsAddr to SOS GPA and append to
-		 * kernel cmdline
-		 */
-		if (boot_params_addr != NULL) {
-			(void)memset(buf, 0U, sizeof(buf));
-			snprintf(buf, MAX_BOOT_PARAMS_LEN, "%s0x%X ",
-				boot_params_arg, hva2gpa(vm, boot_params_addr));
+		(void)memset(buf, 0U, sizeof(buf));
+		status = sbl_seed_parse(vm, cmd_src, buf, sizeof(buf));
```

Comment:
what does this mean? can you add comments? e.g. first attempt to get from sbl, if failure, then try abl?

Reply:
OK, will add comments